test/chunktransformer.ts

```diff
@@ -14,7 +14,7 @@
 
 import * as assert from 'assert';
 import {describe, it, before, beforeEach, afterEach} from 'mocha';
-import * as Long from 'long';
+import Long = require('long');
```

Comment:
why is this import statement changing?

Reply:
IIRC, some compiler or linting error wasn't letting us do it the old way, but we must have made a recent change because it doesn't seem to be a problem anymore so I changed it back to an import.